pkg/service/service.go

```diff
@@ -19,6 +21,23 @@ type ServiceInfo struct {
 	Status string
 }
 
+type ServiceClass struct {
+	Name              string
+	Bindable          bool
+	ShortDescription  string
+	LongDescription   string
+	Tags              []string
+	VersionsAvailable []string
+	ServiceBrokerName string
+}
+
+type ServicePlans struct {
+	Name        string
+	DisplayName string
+	Description string
+	Required    []string
+}
```

Comment:
Perhaps it would be good to add the non required parameters as well. 
That way the user would be aware of the features provided by the service

Reply:
Can you provide where I can get these values?

Reply:
Sure but things might get messed up later. Maybe pick this up in a separate PR?

Reply:
No need. I can just open a PR against yours and if you like it, you merge it into yours and your branch will be up to date - and so will your PR. 
Then once your PR is merged (and squashed), then my commit will not exist :)

Reply:
I have opened a PR against your branch: https://github.com/mik-dass/odo/pull/1

Feel free to give it a spin and let me know what you think

Reply:
OK sounds good :+1:

Reply:
@geoand Done. Thanks for the help, really appreciate it :)

Reply:
I'm glad to help an already great looking PR of yours :wink:

Reply:
@geoand But you made it even better :D

Reply:
A small stone put toward a good foundation :P

Reply:
:D